wiki/Website-Markdown.md

```diff
@@ -191,6 +192,28 @@ Some text content here
 Some other content here
 ```
 
+#### Video
+
+Video assets are not natively supported in markdown, but you can easily insert a .mp4 or similar file type into a markdown with a basic understanding of HTML:
+
+```md
+<video width="100%" controls loop>
+  <source
+    src="assets/{some-subdirectory}/{path-to-video-file}.mp4"
```

Comment:
[blocker] with this guidance we risk someone to put the assets in a random folder; we should instead be prescriptive of where these assets are stored on the website and as far as I can see the path currently used is `/assets/components/{name-of-component}/...` (the `/` in front of the path is required)

```suggestion
    src="/assets/components/{name-of-component}/{path-to-video-file}.mp4"
```

Reply:
There are also some video assets in the pattern documentation, but for simplicity this seems to be the better route

Reply:
Ahh, in that case maybe something like 
`src="/assets/[components|patterns]/{name-of-component}/{path-to-video-file}.mp4"`

Reply:
+1